static/sass/_pattern_cube-table.scss

```diff
@@ -121,6 +133,12 @@
 
     > .p-table--cube__badge {
       align-self: center !important;
+
+      @media only screen and (min-width: $breakpoint-medium) {
+        .is-passed {
```

Comment:
What is this for? It looks like this state class is used in conjunction with `.p-table--cube__badge` and so can not be matched as this would need `&.is-passed`.

Reply:
It looks like you are replacing the code below which is: 
```css
margin-left: -0.2rem;
```

Reply:
Good spot thanks. This is hard to QA as when you pass a microcert a new "passed" orange badge is sent from badgr which has slightly different dimensions. I QA'd it at the time but since the alpha has gone live I've no longer passed any. I will replace this as come to think of it I think I needed this style on all screen sizes.